### PR TITLE
Index.html

### PR DESCRIPTION
**Pull request by Megraohoh**

### CODE REVIEW

**Comment by Megraohoh** [comment]
@alexsaber

**Comment by alexsaber89** [comment]
@Megraohoh, looks good!  Nice styling on the navbar!  One suggestion would be to group your h1's, div's, etc. inside of "container" div's for each section.  Also, there's an extra body tag that can be deleted.  :+1:

File: index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+<header>
+	<nav>
+		<ul>
+			<li class="panic">George Michael's Panic</li>
+			<li class="rebel">Maeby's Rebelliousness</li>
+			<li class="liquor">Lucille's Liquor Cabinet</li>
+		</ul>
+	</nav>
+</header>
+<body>
```
Comment (alexsaber89) [inline index.html:17]: This body tag can be deleted

**Comment by alexsaber89** [comment]
👍

**Pull request merged**